packages/sdk-browser/src/sdk-client.ts

```diff
@@ -92,6 +96,57 @@ export class TurnkeyBrowserSDK {
     });
   };
 
+  // With export and import, we do not call init() so that the caller can control when they'd like to insert the iframe into the DOM
```

Comment:
while we could use a helper (or some mechanism) to manage export / import iframes, this isn't the best route.